test/uxCapture.test.js

```diff
@@ -28,17 +30,16 @@ const onMeasure = jest.fn();
 
 describe('UXCapture', () => {
 	describe('startView', () => {
-		beforeAll(() => {
-			UXCapture.create({ onMark, onMeasure });
-		});
-
 		beforeEach(() => {
 			onMark.mockClear();
 			onMeasure.mockClear();
-
-			UXCapture.startTransition();
```

Comment:
`startTransition` no longer allowed to be called before any view has started

Reply:
Not sure I follow. Can you describe the new sequence of execution here?

Reply:
See the description in README, but still need to wrap my brain around that...

Reply:
The sequence is,
1. Page Load -> call `startView`.
2. Interactive Route transition ->  call `startTransition`, call `clearMarks()` for any marks that need to be removed, call `startView` again. The library knows to look for any existing marks in each zone in the performance timeline but will record new marks for anything that doesn't exist.

Reply:
it's helpful to think of `startTransition` as the signal that the application intends to change from the current view to a new view - i.e. it is transitioning between views. It doesn't apply to non-SPA applications that never transition

Reply:
From what I gathered, essentially you don't want startTransition() called before **first** startView() is called - after that, it's OK.